lottie/src/main/java/com/airbnb/lottie/utils/LottieLogger.java

```diff
@@ -0,0 +1,15 @@
+package com.airbnb.lottie.utils;
+
+/**
+ * Give ability to integrators to provide another logging mechanism.
+ */
+public interface LottieLogger {
```

Comment:
Nothing in the `utils` package will be available to consumers because of [this](https://github.com/airbnb/lottie-android/tree/master/lottie/src/main/java/com/airbnb/lottie/utils). All exposed APIs have to be in the top level package.

Reply:
Will fix